### PR TITLE
Add components and styled carousel module

### PR DESCRIPTION
**Pull request by joSecret**

### CODE REVIEW

File: modules/edw_paragraphs/modules/edw_paragraphs_carousel/components/slider/slider.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+// (($, Drupal) => {
```
Comment (marapredescu1) [inline modules/edw_paragraphs/modules/edw_paragraphs_carousel/components/slider/slider.js]: Comments left here

File: modules/edw_paragraphs/modules/edw_paragraphs_carousel/templates/paragraph--edw-carousel.html.twig [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{#
+/**
+ * @file
+ * UEDW carousel item module implementation to display a paragraph.
```
Comment (marapredescu1) [inline modules/edw_paragraphs/modules/edw_paragraphs_carousel/templates/paragraph--edw-carousel.html.twig]: typo and name

File: modules/edw_paragraphs/modules/edw_paragraphs_carousel/components/slider/slider.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+/**
+ * @file
+ * slider.js
+ */
+
+(function ($, Drupal, drupalSettings, once) {
```
Comment (Anthelixes) [inline modules/edw_paragraphs/modules/edw_paragraphs_carousel/components/slider/slider.js]: `drupalSettings` it's not used. Can we remove it?

File: modules/edw_paragraphs/modules/edw_paragraphs_carousel/templates/paragraph--edw-carousel.html.twig (hunk 2 above)
Comment (joSecret) [reply modules/edw_paragraphs/modules/edw_paragraphs_carousel/templates/paragraph--edw-carousel.html.twig]: Thanks. Fixed.

File: modules/edw_paragraphs/modules/edw_paragraphs_carousel/components/slider/slider.js (hunk 1 above)
Comment (joSecret) [reply modules/edw_paragraphs/modules/edw_paragraphs_carousel/components/slider/slider.js]: Thanks. Fixed.

File: modules/edw_paragraphs/modules/edw_paragraphs_carousel/edw_paragraphs_carousel.libraries.yml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+carousel:
+  css:
+    base:
+      /libraries/slick/slick/slick.css: {}
+  js:
+    /libraries/slick/slick/slick.min.js: { minified: true }
```
Comment (Anthelixes) [inline modules/edw_paragraphs/modules/edw_paragraphs_carousel/edw_paragraphs_carousel.libraries.yml:12]: It's not the same behavior if we include `slick/slick` as a dependency?

File: modules/edw_paragraphs/modules/edw_paragraphs_carousel/components/slider/slider.component.yml [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+slots:
+  slide_slot_main:
+    title: Main
+    description: Main slot description.
+
+libraryOverrides:
+  dependencies:
+    - edw_paragraphs_carousel/carousel
+
```
Comment (Anthelixes) [inline modules/edw_paragraphs/modules/edw_paragraphs_carousel/components/slider/slider.component.yml:46]: `once()` function is used in `slider.js`. I think you should declare `drupal/once` here.

**Pull request merged**